src/librustdoc/html/static/main.js

```diff
@@ -385,6 +386,127 @@
                 }
             }
 
+            function transformResults(results, isType) {
+                var out = [];
+                for (i = 0; i < results.length; ++i) {
+                    if (results[i].id > -1) {
+                        var added = false;
+                        var obj = searchIndex[results[i].id];
+                        obj.lev = results[i].lev;
+                        if (isType !== true || obj.type) {
```

Comment:
Also not really an issue with this PR, but the `isType !== true` in an if thing really isn't saving you any speed of `!isType`.

Reply:
Don't care. In principle it's faster since the comparison is explicit.